src/ko/2018-01/05/01.md

```diff
@@ -1,7 +1,16 @@
 ---
-title:  교훈
+title:  에덴 이후의 청지기들
 date:   27/01/2018
 ---
 
-### <center>우리는이 공과를 위해 노력하고있다..</center>
-<center>나중에 다시 시도 해주십시오..</center>
\ No newline at end of file
+### 연구 범위
+사 22:14~18, 고전 4:1~2, 골 2:2~3, 엡 6:13~17, 고후 5:10
+
+> <p>기 억 절</p>
+“오직하나님께옳게여기심을입어복음을위탁받았으니우리가이와 같이 말함은 사람을 기쁘게 하려 함이 아니요 오직 우리 마음을 감찰하 시는 하나님을 기쁘시게 하려 함이라”(살전 2:4).
```

Comment:
@imasaru Looks like you have same issue as me when I copy certain parts from PDF, it does not copy spaces

Reply:
@VitalikL I fixed this a minute ago... Hopefully we can catch them all.